docs_user/modules/proc_configuring-openshift-worker-nodes.adoc

```diff
@@ -72,3 +72,18 @@ items:
       kubernetes.io/hostname: ocp-worker-0
       node-role.kubernetes.io/worker: ""
 ```
+
+// TODO: Move this to the IPv6 section once it is fully documented, both upstream and downstream.
+ifeval::["{build}" != "downstream"]
+[WARNING]
+In IPv6, {rhocp_long} worker nodes need a `/64` prefix allocation due to OVN
```

Comment:
(Minor point, not sure we would want to capture it) I think it's OVN-IC / OVN-Kubernetes limitation. (Which stems from the fact that SLAAC prefixes are used to allocate addresses.)